### PR TITLE
Install LTS version of nodejs (6.9) by default to from PPA

### LINKED ISSUE #9: Put docs about using this formula in README

### PR DESCRIPTION
**Pull request by ek9**
Install LTS version of nodejs (`6.x`) by default to from PPA instead of no longer maintained `0.12` series.

Update readme on PPA use ( fixes #9 )

### CODE REVIEW

File: node/pkg.sls [hunk 1]
```diff
@@ -8,7 +8,7 @@ nodejs.ppa:
       - pkgrepo: nodejs.ppa
   pkgrepo.managed:
     - humanname: NodeSource Node.js Repository
-    - name: deb {{ salt['pillar.get']('node:ppa:repository_url', 'https://deb.nodesource.com/node_0.12') }} {{ grains['oscodename'] }} main
+    - name: deb {{ salt['pillar.get']('node:ppa:repository_url', 'https://deb.nodesource.com/node_6.x') }} {{ grains['oscodename'] }} main
```
Comment (myoung34) [inline node/pkg.sls]: you should change this to be in map.jinja instead of hardcoding it to prevent having to do this exact PR in the future

**Comment by myoung34** [comment]
@ek9 the changes to the non-documentation parts are useless, no?
Ie. the code hasn't actually changed, and I don't know about others but I typically dont change default values that dont error in case it breaks others who use defaults.

Cut it down to the `README.md` and `pillar.example` and i'll hit merge

**Comment by ek9** [comment]
> Ie. the code hasn't actually changed, and I don't know about others but I typically dont change default values that dont error in case it breaks others who use defaults.

This commit is for people like you, who depend on defaults. Using an unmaitained `0.12` version of node isn't something that should be done by default. I'd understand if it would pull the package from official debian distribution repos, then it wouldn't be a problem. However, as it currently is, this package install no longer maintained version by default, which is an issue on it's own.

Formulas should provide somewhat sensible and meaningful defaults. I dont believe installing a version of node which isn't maintained anymore is one of those things.

**Comment by ek9** [comment]
Thank you.

**Pull request merged**